<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossDownloaderClass</class>
 <widget class="QMainWindow" name="CrossDownloaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>544</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CrossDownloader</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::close-button{\n    border-image: url(:/CrossDownloader/Resources/close.png);\n}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_downloading">
       <attribute name="title">
        <string>视频文件</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="ResourceTableWidget" name="tableWidget">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>文件</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>状态</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>大小</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分辨率</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>评</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>赞</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>分享</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>创建</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addUrl"/>
   <addaction name="monitor"/>
   <addaction name="separator"/>
   <addaction name="setting"/>
   <addaction name="openDownloadFolder"/>
   <addaction name="action_Worker"/>
   <addaction name="extention"/>
  </widget>
  <widget class="QToolBar" name="actionBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="siteBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="douyin"/>
   <addaction name="xiaohongshu"/>
  </widget>
  <action name="addUrl">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/add.png</normaloff>:/CrossDownloader/Resources/add.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="setting">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/setting.png</normaloff>:/CrossDownloader/Resources/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="openDownloadFolder">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/folder-open.png</normaloff>:/CrossDownloader/Resources/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>下载目录</string>
   </property>
  </action>
  <action name="douyin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/douyin.png</normaloff>:/CrossDownloader/Resources/douyin.png</iconset>
   </property>
   <property name="text">
    <string>抖音</string>
   </property>
  </action>
  <action name="extention">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/chrome.png</normaloff>:/CrossDownloader/Resources/chrome.png</iconset>
   </property>
   <property name="text">
    <string>插件</string>
   </property>
  </action>
  <action name="monitor">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/live.png</normaloff>:/CrossDownloader/Resources/live.png</iconset>
   </property>
   <property name="text">
    <string>直播监控</string>
   </property>
  </action>
  <action name="xiaohongshu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/xhs.png</normaloff>:/CrossDownloader/Resources/xhs.png</iconset>
   </property>
   <property name="text">
    <string>小红书</string>
   </property>
  </action>
  <action name="action_Worker">
   <property name="icon">
    <iconset resource="crossdownloader.qrc">
     <normaloff>:/CrossDownloader/Resources/workershow.png</normaloff>:/CrossDownloader/Resources/workershow.png</iconset>
   </property>
   <property name="text">
    <string>Worker</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ResourceTableWidget</class>
   <extends>QTableWidget</extends>
   <header>resourcetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="crossdownloader.qrc"/>
 </resources>
 <connections/>
</ui>
